src/ChainSafe.Gaming.WebPageWallet/WebPageWallet.cs

```diff
@@ -13,14 +16,49 @@
 using Nethereum.Signer;
 using Nethereum.Util;
 using Newtonsoft.Json;
+using WalletConnectSharp.Sign.Models;
+using WalletConnectSharp.Sign.Models.Engine;
 using AddressExtensions = ChainSafe.Gaming.Web3.Core.Debug.AddressExtensions;
 
 namespace ChainSafe.Gaming.Wallets
 {
     public class WebPageWallet : ISigner, ITransactionExecutor, ILifecycleParticipant
     {
+        public WebPageWallet(IRpcProvider provider, WebPageWalletConfig configuration, IOperatingSystemMediator operatingSystem, IChainConfig chainConfig)
```

Comment:
You should merge this class & WalletConnectUnity into WalletConnectWallet, remove all WebPageWallet logic that you don't need out of it and put it into a separate project.
We can keep WebPageWallet for a couple of weeks I guess and remove it when we decide that the time has come.

Reply:
I'm not completely sure about this though since `SendTransaction` still uses webpage redirection and clipboards

Reply:
Oh. Can we implement it using just WalletConnect, so that there are no clipboards?

Reply:
it's doable but for now we can't @kalambet mentioned that transactions need to be handled through ChainSafe since there's extra processing happening on the backend, for example for Gelato Gasless, so we can't use Wallet Connect for send transaction until we can call the two separately, which needs backend work.